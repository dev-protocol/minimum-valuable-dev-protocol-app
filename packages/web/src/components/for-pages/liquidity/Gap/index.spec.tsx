import React from 'react'
import { render } from '@testing-library/react'
import { Gap } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/pages/liquidity/fixtures/geyser/hooks.ts')
jest.mock('src/pages/liquidity/fixtures/uniswap-pool/hooks.ts')

describe(`${Gap.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Gap></Gap>)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
