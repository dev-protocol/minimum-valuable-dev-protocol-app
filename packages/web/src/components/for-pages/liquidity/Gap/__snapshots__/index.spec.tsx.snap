// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`undefined Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-4swyw2-0 lpiJWU"
    />
  </div>
</body>
`;
